src/nginx/t/auth_asymmetrickey.t

```diff
@@ -119,33 +125,36 @@ my $rs256_pubkey = <<'EOF';
 }
 EOF
 
+# es256_token is issued by "test-esp-auth.com".
```

Comment:
add comment to describe ways to generate this token
same for rs256 token

Reply:
Done.